### ISSUE TITLE
Java Editor (in 1.80) graphic glitches

### ISSUE THREAD

**Issue by korakios** [monkeyswarm/MobMuPlatEditor#12]
Hi,
I used 1.74 with no problems ,but in 1.80 there are redraw issues with some patches and others do not show at all (like MMPExamples-Motion-Pad.mmp). The problem is not only in the area of the virtual screen ,but in the also Editor (the menus not showing)
Tested both Windows (10x64bit ,Java Version 8 Update 111) and Linux (custom Debian) .

**Comment by monkeyswarm** [comment]
Patched and re-uploaded. 
Please re-download the development package from the webpage 
(MobMuPlatDistribution_1.80iOS_0.30Android.zip)
And try the java editor again. Thanks.

**Comment by korakios** [comment]
Sorry for the delay, I work on a project and for failsafe reasons I continued on 1.74.
Tried the 1.80 version ,there are still problems. Try the "MMPTutorial1-GUI-Pad.mmp" and the last page (3) is not shown until clicking on the virtual screen and some glitched graphics of the page show-up.
On a progect I work on 1.78 ,the redraw issue is still present all over the 1.80 editor.

**Comment by monkeyswarm** [comment]
I re-tested on current OSX and Ubuntu and don't see the issue. Are you sure you re-downloaded the current version? I update the editor but I did not change the version numbering.

**Comment by korakios** [comment]
You are right ,I've messed the versions, redownloaded and it works. A minor glitch is shown in the picture but it's nothing serious
![mmp](https://cloud.githubusercontent.com/assets/7697015/21934740/402b40c0-d9b3-11e6-8342-3a2c6a4fd10b.png)
However I still have problem opening a project (a synth editor). I worked with the older editor and in the new one I get white screen and some graphics are updated when clicking blindly with the mouse
[C41c.zip](https://github.com/monkeyswarm/MobMuPlatEditor/files/704626/C41c.zip)

**Comment by korakios** [comment]
On Ubuntu 16.04 I opened the Java Editor on terminal:
This is the output:

[MMP_ubuntu1604_console_output.txt](https://github.com/monkeyswarm/MobMuPlatEditor/files/704718/MMP_ubuntu1604_console_output.txt)

PS. Hope you find time for posting about the 'sysex dump', and 'set page' issue :)
